src/pages/components/variants/inputCheckbox/WithError.tsx

```diff
@@ -1,13 +1,62 @@
+import { ChangeEvent, useEffect, useState } from 'react';
 import { InputCheckbox } from '@boston-scientific/anatomy-react';
+import { Fieldset } from '@boston-scientific/anatomy-react';
 import Example from 'shared/components/Example';
 
 const WithError = (): JSX.Element => {
+  const errorMessage = 'This is an example of an error message.';
+  const [errorText, setErrorText] = useState(errorMessage);
+  const [checkboxes, setCheckboxes] = useState([
+    {
+      text: 'Checkbox 1',
+      isChecked: true
+    },
+    {
+      text: 'Checkbox 2',
+      isChecked: false
+    },
+    {
+      text: 'Checkbox 3',
+      isChecked: false
+    }
+  ]);
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>, index: number) => {
+    const updatedCheckboxes = [...checkboxes];
+    updatedCheckboxes[index].isChecked = e.target.checked;
+    setCheckboxes(updatedCheckboxes);
+  };
+
+  useEffect(() => {
+    if (checkboxes.filter((c) => c.isChecked === true).length < 2) {
+      setErrorText(errorMessage);
+    } else {
+      setErrorText('');
+    }
+  }, [checkboxes]);
+
   return (
-    <Example>
-      <div className="bsds-form-control">
-        <InputCheckbox label="Checkbox" forceValidation required />
-      </div>
-    </Example>
+    <>
+      <Example>
```

Comment:
Do we need a ticket to update docs to reflect the changes to validation logic here? Right now we have "check the checkbox in the preview to clear the error" but one of the checkboxes is already checked in the group. 

Currently checking/unchecking the single checkbox toggles the two different error messages.

Reply:
the checkbox group required two checkboxes to be checked to clear the error - lmk if you want to keep this as is.